packages/react-popover/e2e/support.js

```diff
@@ -1,2 +0,0 @@
-// workaround for https://github.com/cypress-io/cypress/issues/8599
```

Comment:
shouldn't we keep those for consistency? if someone would like to add some only for particular package etc

Reply:
no these were never needed. In fact the support script could have always been configured from the `cypress.js` file in scripts. IMO this was a bad way of doing this because there was no support commands that are used anywhere

Reply:
If we have the requirement to use specific support commands we can reconsider.